src/sidebar/views/overview.ts

```diff
@@ -16,91 +16,87 @@ export class OverviewItem {
+        'loading': [
+            new OverviewItem('Loading overview, please wait...')
+        ],
+        'notFound': [
+            new OverviewItem('CodeChecker run not found.'),
+            new OverviewItem('Run CodeChecker, reload metadata,'),
+            new OverviewItem('or set the output folder to get started')
+        ],
+        'normal': [
+            new OverviewItem(() => `Total files analyzed: ${ExtensionApi.metadata.sourceFiles.size}`),
+            new OverviewItem(() => `Last analysis run: ${
+                new Date(ExtensionApi.metadata.metadata!.timestamps.begin /* seconds */ * 1000).toLocaleString()
+            }`),
+            new OverviewItem(() => {
+                // Time in seconds
+                const beginTime = ExtensionApi.metadata.metadata!.timestamps.begin;
+                const endTime = ExtensionApi.metadata.metadata!.timestamps.end;
+                const interval = endTime - beginTime;
+
+                const hours = Math.floor(interval/3600) % 60;
+                const minutes = Math.floor(interval/60) % 60;
+                const seconds = Math.floor(interval) % 60;
+                const ms = Math.floor(interval * 1000) % 1000;
+
+                if (hours > 0 || minutes > 0) {
+                    // H:MM:SS s / M:SS s
+                    const formattedMinutes = hours > 0 ? (`${hours}:${minutes.toString().padStart(2, '0')}`) : minutes;
+                    return `Analysis duration: ${formattedMinutes}:${seconds.toString().padStart(2, '0')} s`;
+                } else {
+                    // S.fff s
+                    return `Analysis duration: ${(seconds + ms / 1000).toFixed(3)} s`;
+                }
+            }),
+            new OverviewItem(() => `Used analyzers: ${
+                Object.keys(ExtensionApi.metadata.metadata!.analyzers).join(', ')
+            }`),
+        ]
     };
 
-    private regularItemsList = [
-        'files',
-        'lastRun',
-        'buildLength',
-        'analyzers',
-
-        'separator',
-
-        'reloadMetadata',
-        'analyzeCurrentFile',
-        'analyzeProject',
-    ];
-    private notFoundItemsList = [
-        'notfound',
-        'notfound2',
-        'notfound3',
+    private bottomItems: {[id: string]: OverviewItem[]} = {
+        'normal': [
+            new OverviewItem('Reload CodeChecker metadata', {
+                title: 'reloadMetadata',
+                command: 'codechecker.backend.reloadMetadata',
+            }),
+            new OverviewItem('Re-analyze current file', {
+                title: 'reloadMetadata',
+                command: 'codechecker.executor.analyzeCurrentFile',
+            }),
+            new OverviewItem('Re-analyze entire project', {
+                title: 'reloadMetadata',
+                command: 'codechecker.executor.analyzeProject',
+            }),
+        ],
+        'ccNotFound': [
+            new OverviewItem('Reload CodeChecker metadata', {
```

Comment:
I clicked on the  `Show database setup dialog...` item, initialized a compilation database but the CodeChecker view still shows me that the compilation database is not shown even if I click on the File view and go back to the CodeChecker view:
![image](https://user-images.githubusercontent.com/6695818/141768684-3a2c283d-2522-4ac2-84e2-4c2f26e2bff9.png)

If I click on the Reload CodeChecker metadata button the view will be refreshed and it will show me the following buttons:
![image](https://user-images.githubusercontent.com/6695818/141769032-42fa4e00-a111-4ec1-85b2-a9a131d7cac2.png)

Can not we show this automatically.

Reply:
Added watches, so the sidebar should update automatically even when running Log.